app/assets/javascripts/analytics/ecommerce.js

```diff
@@ -0,0 +1,62 @@
+(function () {
+  "use strict";
+  window.GOVUK = window.GOVUK || {};
+
+  var Ecommerce = function (config) {
+    this.init = function (element) {
+      var self = this;
+      var ecommerceRows = element.find('[data-ecommerce-row]');
+      var startPosition = parseInt(element.data('ecommerce-start-index'));
```

Comment:
Doesn't calling `.data` coerce the value into an integer? (`parseInt` should also have a base `10` passed in as the 2nd argument – eg if the HTML has the value "09" then this would be parsed as an incorrect number on old browsers.